network/network.go

```diff
@@ -32,32 +32,32 @@ type SyncStream interface {
 // Network represents the behavior of the network
 type Network interface {
 	// Broadcast propagates a signed message to all peers
-	Broadcast(msg *proto.SignedMessage) error
+	Broadcast(topicName []byte, msg *proto.SignedMessage) error
```

Comment:
@nivBlox why did we add topicName?

Reply:
we used the pubkey from the msg but now that we removed it we need to pass it in order to fetch the topic